modules/modules.mk

```diff
@@ -31,55 +31,60 @@ ifeq ($(ALL_MODULES),yes)
         POROUS_FLOW                 := yes
         RAY_TRACING                 := yes
         RDG                         := yes
-				REACTOR                     := yes
+        REACTOR                     := yes
         RICHARDS                    := yes
         STOCHASTIC_TOOLS            := yes
         TENSOR_MECHANICS            := yes
+        THM                         := yes
         XFEM                        := yes
 endif
 
+# Modules that follow have one or more dependencies
+# on the modules defined above them.
+
 ifeq ($(CONTACT),yes)
         TENSOR_MECHANICS            := yes
 endif
 
 ifeq ($(FSI),yes)
-			  NAVIER_STOKES               := yes
+        NAVIER_STOKES               := yes
         TENSOR_MECHANICS            := yes
 endif
 
+ifeq ($(HEAT_CONDUCTION),yes)
+        RAY_TRACING                 := yes
+endif
+
 ifeq ($(PERIDYNAMICS),yes)
         TENSOR_MECHANICS           := yes
 endif
 
+ifeq ($(PHASE_FIELD),yes)
+        TENSOR_MECHANICS            := yes
+endif
+
 ifeq ($(POROUS_FLOW),yes)
         CHEMICAL_REACTIONS          := yes
         FLUID_PROPERTIES            := yes
         TENSOR_MECHANICS            := yes
 endif
 
-ifeq ($(XFEM),yes)
-        TENSOR_MECHANICS            := yes
+ifeq ($(NAVIER_STOKES),yes)
+        FLUID_PROPERTIES            := yes
+        HEAT_CONDUCTION             := yes
+        RDG                         := yes
 endif
 
-# Modules that follow have one or more dependencies
```

Comment:
Why remove this comment? Breaking the point where the includes aren't independent anymore I think is good